java/client/src/main/java/glide/api/models/commands/RangeOptions.java

```diff
@@ -110,6 +172,22 @@ public String toArgs() {
         }
     }
 
+    /**
+     * Enumeration representing lexicographic positive and negative infinity bounds for sorted set.
+     */
+    @RequiredArgsConstructor
+    public enum InfLexBoundBinary implements LexRangeBinary {
```

Comment:
If a class does not really uses binary data and only uses hard-coded strings that are always valid UTF8 strings, we could simply add `toGlideStringArgs()` and avoid duplicating the code (we did this in multiple locations)